{"uuid": "4f96932f-4b25-40fe-847a-6edff31f5282", "children": ["c056a4e8-f415-47e9-8e03-432c639bc157"], "befores": [{"name": "mobile_management", "status": "passed", "start": 1721578312799, "stop": 1721578314958}], "afters": [{"name": "mobile_management::0", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=54825): Max retries exceeded with url: /session/a24917f3e0040a6c6d71de3060e99c41/screenshot (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x1045e24b0>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "trace": "  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 906, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/tests/conftest.py\", line 59, in mobile_management\n    utils.allure.add_screenshot()\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/wikipedia_tests/utils/allure.py\", line 8, in add_screenshot\n    png = browser.driver.get_screenshot_as_png()\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 352, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/remote_connection.py\", line 302, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/selenium/webdriver/remote/remote_connection.py\", line 322, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/_request_methods.py\", line 136, in request\n    return self.request_encode_url(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/_request_methods.py\", line 183, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/connectionpool.py\", line 873, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/connectionpool.py\", line 873, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/connectionpool.py\", line 873, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/connectionpool.py\", line 843, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/Users/torsukov/Documents/projects/qa-guru/wikipedia-tests/.venv/lib/python3.12/site-packages/urllib3/util/retry.py\", line 519, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1721578320289, "stop": 1721578320299}], "start": 1721578312799, "stop": 1721578320299}